x/distribution/keeper/grpc_query.go

```diff
@@ -19,8 +19,7 @@ var _ types.QueryServer = Keeper{}
 // Params queries params of distribution module
 func (k Keeper) Params(c context.Context, req *types.QueryParamsRequest) (*types.QueryParamsResponse, error) {
```

Comment:
nit: we should create an issue to move all gRPC methods to a dedicated querier wrapper type around the keeper :)

Reply:
created: https://github.com/cosmos/cosmos-sdk/issues/12465